OpenEdXMobile/src/main/java/org/edx/mobile/view/adapters/CourseUnitPagerAdapter.java

```diff
@@ -63,7 +64,11 @@ public Fragment getItem(int pos) {
         if (isCourseUnitVideo(unit)) {
             unitFragment = CourseUnitVideoFragment.newInstance((VideoBlockModel) unit, (pos < unitList.size()), (pos > 0));
         } else if (unit instanceof VideoBlockModel && ((VideoBlockModel) unit).getData().encodedVideos.getYoutubeVideoInfo() != null) {
-            unitFragment = CourseUnitOnlyOnYoutubeFragment.newInstance(unit);
+            if (config.getYoutubeConfig().isYoutubeEnabled()) {
```

Comment:
I think maybe it's better to remove the indented if statement here and mimic what happens on line 72 for discussions. Might mean re-ordering the if clauses

```
if isCourseUnitVideo
else if youtube enabled and is youtube
else if is youtube
else if …
```

Reply:
ok